### PR TITLE
[Look and Feel] CAT Home Page

### PR DESCRIPTION
**Pull request by MichaelCherry**
# Description

Updating the CAT home page to match the look and feel of https://trello.com/c/AnV6sJuG/28-build-cat-home-page

## Type of change

Please delete options that are not relevant.

- Code cleanliness or refactor

## Screenshot

Before:
image

After:
image

# Testing

Applicable for all code changes.

Manual steps to reproduce this functionality:

1.  Go to Home page.
2.  Verify that the home page looks correct

Screenshot of unit tests run passing:
image

# Checklist

Applicable for all code changes.

- [x] My code follows the style guidelines of this project
- [ ] ~~I have commented my code, particularly in hard-to-understand areas~~
- [ ] ~~I have made corresponding changes to the documentation~~
- [x] My changes generate no new compiler warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have researched WCAG2.1 accessibility standards and met them in this PR (can be navigated using a keyboard)
- [x] My changes look good on IE 10+, ~~Firefox~~, and Chrome

### CODE REVIEW

File: frontend/src/Home.jsx [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
+import big_welcome_banner from "./images/big_welcome_banner.png";
 import LOCALIZE from "./text_resources";
 
 const Home = () => {
   return (
     <div className="app">
-      <h2>{LOCALIZE.homePage.title}</h2>
-      <p>{LOCALIZE.homePage.welcomeMsg}</p>
-      <img src={logo} alt="logo" />
+      <h2 id="home-page-paragraph">{LOCALIZE.homePage.title}</h2>
```
Comment (caleybrock) [inline frontend/src/Home.jsx]: Since we don't have an H1 on the page yet, I think this needs to be an h1

Comment (MichaelCherry) [reply frontend/src/Home.jsx]: resolved in eba175e

**Comment by MichaelCherry** [comment]
Resolved merge conflict;
Tests still pass, etc:

image

**Comment by fnormand01** [comment]
It looks awesome! However, we'll need to updated the translation in the background image.
**_Bienvenue au Competency Assessment Tool_** should be more like _**Bienvenue dans l'outil d'évaluation des compétences**_.

But for now, this look great!

**Pull request merged**